src/App.js

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import './App.css';
 import { Container } from 'semantic-ui-react';
 import Map from './component/map/Map.js';
+import HelpButton from './component/help/HelpButton.js';
 
 function App() {
   return (
-    <Container>
-      <Map/>
-    </Container>
+    <React.Fragment>
+      <HelpButton />
```

Comment:
Had to put it outside the container so it would stay in the corner; the Container component adds margins.